src/ninja.cc

```diff
@@ -223,18 +223,19 @@ void Usage(const BuildConfig& config) {
 "  -v, --verbose  show all command lines while building\n"
 "  --quiet        don't show progress status, just command output\n"
 "\n"
-"  -C DIR   change to DIR before doing anything else\n"
-"  -f FILE  specify input build file [default=build.ninja]\n"
+"  -C DIR         change to DIR before doing anything else\n"
+"  -f FILE        specify input build file [default=build.ninja]\n"
 "\n"
-"  -j N     run N jobs in parallel (0 means infinity) [default=%d on this system]\n"
-"  -k N     keep going until N jobs fail (0 means infinity) [default=1]\n"
-"  -l N     do not start new jobs if the load average is greater than N\n"
-"  -n       dry run (don't run commands but act like they succeeded)\n"
+"  -j N           run N jobs in parallel (0 means infinity) [default=%d on this system]\n"
+"  -k N           keep going until N jobs fail (0 means infinity) [default=1]\n"
+"  -l N           do not start new jobs if the load average is greater than N\n"
+"  -n             dry run (don't run commands but act like they succeeded)\n"
+"  -p PRIORITY    set the priority/niceness of spawned processes\n"
```

Comment:
Please document possible values and their meaning + perform range checks in the parsing code below.

Reply:
Isn't this highly dependent on your scheduler implementation? On Linux this varies based on the scheduling policy -- in Android builds we run ninja under `SCHED_BATCH`, which doesn't allow changing this.

It looks like the range checks would need to use `sched_get_priority_min` and `sched_get_priority_max` to check the current policy.

Reply:
Since this is so platform and scheduler dependent, should I still do the range checks?